src/components/BigView.jsx

```diff
@@ -0,0 +1,36 @@
+import { useSelector } from 'react-redux';
+import BigViewItem from './BigViewItem';
+import Navbar from './Navbar';
+
+const BigView = () => {
+  const finances = useSelector((store) => store.finance.finance);
+  const isLoading = useSelector((store) => store.finance.isLoading);
+
+  if (isLoading) {
+    return <h1>Loading ... </h1>;
+  }
+
+  return (
+    <div className="container">
+      <Navbar view="Balence view" year={2015} />
+      <div className="row-40">
+        <div className="right-flex">{}</div>
+        <div className="left-flex">
+          <h2>Balance Sheet Statement</h2>
```

Comment:
The Home page should have a list of categories that could be filtered using given parameters, e.g. category name.
### For Example:
You can add a search bar where you can filter out the balance sheet statements by any name or the given date. 👍

Reply:
This change has been done well. Great work adding filter by date